### ISSUE TITLE
Layout.navigation ignores styles

### ISSUE THREAD

**Issue by PublicParadise** [debois/elm-mdl#231]
[Current implementation](https://github.com/debois/elm-mdl/blob/v7/src/Material/Layout.elm#L477-L479) looks like this:
```elm
navigation : List (Options.Style m) -> List (Html m) -> Html m
navigation styles contents =
  nav [class "mdl-navigation"] contents
```

But `styles` is ignored. I was expecting something along the lines of:
```elm
navigation : List (Options.Style m) -> List (Html m) -> Html m
navigation styles contents =
  Options.nav (cs "mdl-navigation" :: styles) contents
```

**Comment by debois** [comment]
Right. Could you submit a PR?

**Comment by bparadie** [comment]
On your way!

**Comment by vipentti** [comment]
This was fixed in #232. Thank you!
